Packages/Toolkit/Routines/XINDX10.m

```diff
@@ -72,6 +72,28 @@ S INP(10)="NAMESPACE",DA=1
  Q:'$$HDR
 ID S INDID=-1 F G=0:0 S INDID=$O(^DD(INDFN,0,"ID",INDID)) Q:INDID=""  I $D(^(INDID))#2 S INDC="ID"_INDID_" ; IDENTIFIER CODE FOR "_INDID S INDX=$S(^(INDID)]"":^(INDID),1:"Q") D ADD
 W I $D(^DD(INDFN,0,"W"))#2 S INDX=^("W"),INDC="W ; 'W' code ??" D ADD
+FILE ;Get additional File level fields that contain executable code
+ I $D(^DD(INDFN,0,"ACT"))#2 S INDC="ACT ; POST-ACTION",INDX=^("ACT") D ADD
+ I $D(^DD(INDFN,0,"DIC"))#2 S INDC="DIC ; SPECIAL LOOKUP",INDX="D ^"_^("DIC") D ADD
+INDX ;Get New-Style Cross-Reference stored in the INDEX File
+ ;We can get this from the "BB" index on the INDEX file (INDEL is the index name)
+ S INDEL="" F  S INDEL=$O(^DD("IX","BB",INDFN,INDEL)) Q:INDEL=""  D
+ . S X=$Q(^(INDEL)),X=$QS(X,5)
```

Comment:
Put the word naked reference in a comment.

Reply:
Done.